_i18n/de/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Eine Open Source Firmware für auf AVR oder ARM basierende Tastaturen"
+permalink: /de/
+lang: "de"
+---
+QMK ist eine auf der [tmk\_keyboard firmware](http://github.com/tmk/tmk_keyboard) basierende Tastatur-Firmware mit einigen nützlichen Zusatzfunktionen für Atmel AVR- sowie ARM-Controller, insbesondere [Produkte von OLKB](http://olkb.com), das [ErgoDox EZ](http://www.ergodox-ez.com) sowie [Produkte von Clueboard](http://clueboard.co/).
+
+## Tastaturen mit QMK
+
+* [Planck](https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/)
+* [Preonic](https://github.com/qmk/qmk_firmware/blob/master/keyboards/preonic/)
+* [ErgoDox EZ](https://github.com/qmk/qmk_firmware/blob/master/keyboards/ergodox_ez/)
+* [Clueboard](https://github.com/qmk/qmk_firmware/blob/master/keyboards/clueboard/)
+* [Cluepad](https://github.com/qmk/qmk_firmware/blob/master/keyboards/clueboard/17/)
+* [Gergo](https://qmk.fm/keyboards/gergo/)
+* [Georgi](https://qmk.fm/keyboards/georgi/)
+* [Atreus](https://github.com/qmk/qmk_firmware/blob/master/keyboards/atreus/)
+
+Diese Projekte verwenden QMK als ihre offizielle Firmware - fehlt Ihres? [Lassen Sie es uns wissen (vorzugsweise auf Englisch)!](https://github.com/qmk/qmk.fm/issues/new)
+
+QMK ist kompatibel mit [vielen anderen Tastaturen und Projekten, welche von der Community unterstützt werden](/keyboards/).
+
+Erfahren Sie mehr über die Verwendung von QMK mit Ihrer Tastatur [hier](/powered/).
+
+## Entwickler
+
+QMK wird von Jack Humbert entwickelt und erhalten, mit Hilfe der QMK-Community und natürlich [Hasu](https://github.com/tmk). Die OLKB-Firmware wird von [Jack Humbert](https://github.com/jackhumbert) entwickelt, die Ergodox EZ-Firmware von [ZSA Technology Labs](https://github.com/zsa) und die Clueboard-Firmware von [Zach White](https://github.com/skullydazed).
+
+## Informationen
+
+Die [Dokumentation](https://docs.qmk.fm) ist auf [Gitbook](https://www.gitbook.com/book/qmk/firmware/details) und [Github](https://github.com/qmk/qmk_firmware/tree/master/docs) zu finden (beide sind identisch). Änderungen können über eine Fork und [Pull Request](https://github.com/qmk/qmk_firmware/pulls) oder über den "suggest an edit"-Knopf auf jeder Seite vorgeschlagen werden.
```

Comment:
The Gitbook link is not working at the moment, it will redirect to https://qmk-1.gitbook.io/firmware .. removing the -1 will redirect to the beta documentation. This affects the english site as well.

Reply:
For now I'll just repeat the first link.

Reply:
Sorry, yeah this needs to be updated in the other one as well. We can handle that separately.